Design.md

```diff
@@ -174,6 +174,15 @@ Originally discussed in [#12](https://github.com/coreos/fedora-coreos-tracker/is
  - For the short term, if we need to include an agent we will bake it into the image. We will not have any specific
    mechanism for including agents.
 
+### Azure:
+
+Originally discussed in [#65](https://github.com/coreos/fedora-coreos-tracker/issues/65).
+
+- We've identified one major gap with not shipping the [Microsoft Azure Linux Agent](https://github.com/Azure/WALinuxAgent): the machine will not check-in and will eventually be culled by Azure for being stuck in the creation process.
+- This gap will be temporarily worked around via a [systemd unit](https://github.com/coreos/fedora-coreos-tracker/issues/65#issuecomment-434449560) which will perform the check-in.
+- Long term this gap will be covered by work done in [coreos-metadata](https://github.com/coreos/coreos-metadata#120).
+- One additional gap which will not be covered is a lack of ephemeral disk support. We plan to ship udev rules but will not have a service which formats the disk unless we receive feature requests in the future.
```

Comment:
can you link to the issue where we discussed this gap and the decision to not try to address it for now